doc/man7/EVP_PKEY-DH.pod

```diff
@@ -17,15 +17,10 @@ See L<EVP_PKEY-FFC(7)> for more information about FFC keys.
 
 The B<DH> key type uses PKCS#3 format which saves p and g, but not the 'q' value.
 The B<DHX> key type uses X9.42 format which saves the value of 'q' and this
-must be used for FIPS186-4.
+must be used for FIPS186-4. If key validation is required users should be aware
```

Comment:
You've not been consistent with FIPS186-4, you've also written it FIPS-186-4.  Personally, I'd use FIPS 186-4 but consistency is the way to go.

Reply:
Will use FIPS186-4 as this is consistent with other places (including SP800-)